labgrid/util/agents/network_interface.py

```diff
@@ -0,0 +1,359 @@
+import sys
+import uuid
+import socket
+from time import monotonic, sleep
+
+import gi
+gi.require_version('NM', '1.0')
+from gi.repository import GLib, Gio, NM
```

Comment:
Does it make sense to add `pygobject` to one of the requirements.txt files?

Reply:
No, as it cannot be installed in the virtualenv and must come from the host.